jupyterlab-dataops/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+mkdir -p /home/jovyan/.snowsql
```

Comment:
this is snowflake related so it should be under snowsql I think.

Reply:
dd1ffe5a7df855f89af33388392c2d7b2c566398